pkgs/development/python-modules/prometheus-flask-exporter/default.nix

```diff
@@ -0,0 +1,21 @@
+{ buildPythonPackage, fetchPypi, flask, lib, prometheus_client }:
+
+buildPythonPackage rec {
+  pname = "prometheus-flask-exporter";
+  version = "0.18.1";
+
+  src = fetchPypi {
+    pname = "prometheus_flask_exporter";
```

Comment:
Unfortunately, the package was uploaded with underscores to Pypi so I had to override this package name.

https://github.com/rycus86/prometheus_flask_exporter/blob/master/setup.py#L17

Reply:
Thats totally fine.